keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -842,15 +842,14 @@ enhancement:
 ### Feature enablement and rollback
 
 * **How can this feature be enabled / disabled in a live cluster?**
-  - [X] Feature gate
-    - Feature gate name: CSIStorageCapacity
-    - Components depending on the feature gate:
-      - apiserver
+  - [X] CSIDriver.StorageCapacity field can be modified
```

Comment:
Please don't change it.

Even though feature gate is locked in GA, we want to keep it here for tracking purposes.

Please add the new info about the field as an addition.

Reply:
Done.